.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "vendor/assets/stylesheets/pygments-css"]
 	path = vendor/assets/stylesheets/pygments-css
 	url = https://github.com/richleland/pygments-css.git
+[submodule "vendor/assets/javascripts/recorderjs"]
```

Comment:
Would it be better to add this to webpack instead?

Reply:
Hence, I need to clone this into the ```app``` directory? Which directory should I clone to ?

Reply:
I think it make sense to move it to webpack, because it's the webpack assets that requires it.
How about create a `client/vendor` folder and clone the lib into it?

Reply:
OK, let me move to webpack instead.